web/src/compose.js

```diff
@@ -482,7 +483,6 @@ export function initialize() {
             event.preventDefault();
             const {$banner_container} = get_input_info(event);
             compose_validate.clear_wildcard_warnings($banner_container);
-            compose_validate.set_user_acknowledged_wildcard_flag(true);
             finish();
```

Comment:
So, this, as written, makes clicking the usual "send" button send/save the message, rather than the previous behavior of requiring you to actually click the "Yes, send" version implemented here.

I believe this is easy to fix without the previous complexity of that global variable: just push a parameter `{acknowledged_wildcard_mention_warning: true}` here and plumb it through.

That said, if you're going to do that, I'm not sure I understand the function of methods like `add_wildcard_banner_shown_for_container` -- ultimately we just need to know which button was pushed to decide whether to actually send the message or not.